atom/browser/native_window_views.cc

```diff
@@ -603,7 +603,8 @@ void NativeWindowViews::SetResizable(bool resizable) {
   if (has_frame() && thick_frame_)
     FlipWindowStyle(GetAcceleratedWidget(), resizable, WS_THICKFRAME);
 #endif
-
+  if (maximizable_)
+    SetMaximizable(maximizable_);
```

Comment:
Should the inverse with `!maximizable_` be true too? ie should the `if (maximizable_)` conditional be removed, calling `SetMaximizable(maximizable_)` unconditionally?

Reply:
Yea we can